certify-mosipid-identity.properties

```diff
@@ -63,9 +64,9 @@ mosip.certify.key-values={\
                                   'postalCode': { 'display': {{'name': 'Postal Code', 'locale': 'en' }}}\
                                }\
                           },\
-                          'display': {{'name': 'MOSIP National ID', \
+                          'display': {{'name': 'Veridonia National ID', \
```

Comment:
Why are we changing this ? Can we not keep MOSIP as MOSIP National Id and use the Veridonia Name to the mock identity usecase ?

Reply:
Changed name to MOSIP for mosip national id